core/new-gui/src/app/dashboard/service/workflow-executions/workflow-executions.service.ts

```diff
@@ -27,4 +28,23 @@ export class WorkflowExecutionsService {
       isBookmarked,
     });
   }
+
+  deleteWorkflowExecutions(wid: number, eId: number): Observable<Object> {
+    return this.http.put(`${WORKFLOW_EXECUTIONS_API_BASE_URL}/delete_execution`, {
+      wid,
+      eId,
+    });
+  }
+
+  updateWorkflowExecutionsName(wid: number | undefined, eid: number, executionName: string): Observable<Response> {
+    return this.http
+      .post<Response>(`${WORKFLOW_EXECUTIONS_API_BASE_URL}/update_name_${wid}_${eid}_${executionName}`, null)
+      .pipe(
+        catchError((error: unknown) => {
+          // @ts-ignore // TODO: fix this with notification component
```

Comment:
please remove the @ts-ignore. instead of throwing error, maybe handle it in some way for example, not update the entry?

Reply:
has been removed